test/new-e2e/go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/DataDog/datadog-agent/test/new-e2e
 
-go 1.21.0
+go 1.22.0
```

Comment:
I don't see this change in the PR you mentioned 🤔 Is it expected?

Reply:
hmm no I guess not